packages/docusaurus-theme-common/src/utils/unlistedUtils.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Translate from '@docusaurus/Translate';
+import Head from '@docusaurus/Head';
+
+export function UnlistedBannerTitle(): JSX.Element {
+  return (
+    <Translate
+      id="theme.unlistedContent.title"
+      description="The unlisted content banner title">
+      Unlisted content
+    </Translate>
+  );
+}
+
+export function UnlistedBannerMessage(): JSX.Element {
+  return (
+    <Translate
+      id="theme.unlistedContent.message"
+      description="The unlisted content banner message">
+      This page is unlisted and can only be accessed directly.
+    </Translate>
+  );
+}
```

Comment:
Any opinion what we should use as a default message?

The title looks fine but the message text is probably not ideal 😅 

<img width="888" alt="CleanShot 2022-10-28 at 12 44 06@2x" src="https://user-images.githubusercontent.com/749374/198569365-81e56798-0679-4294-895e-aefdf60fc6a2.png">

Reply:
I would use word related with "visibility", something like this: 

 > This page won't be seen by other users therefore accessible via direct link only.